src/ObjCRuntime/TrampolineBlockBase.cs

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Runtime.InteropServices;
+
+namespace ObjCRuntime {
+
+	public abstract class TrampolineBlockBase {
```

Comment:
```
ObjCRuntime/TrampolineBlockBase.cs(10,3): error CS0592: Attribute 'DebuggerBrowsable' is not valid on this declaration type. It is only valid on 'property, indexer, field' declarations.
```

Reply:
Huh, today I learned something new.

Reply:
I was surprised too :) but I think you meant

```sharp
[EditorBrowsable (EditorBrowsableState.Never)]
```
since it's _kind of_ what I read, i.e. the intent is that it should not show up in the IDE since it's infrastructure code.